components/messenger.rst

```diff
@@ -175,9 +175,13 @@ Hence you can inspect the envelope content and its stamps, or add any::
     }
 
 The above example will forward the message to the next middleware with an additional
-stamp *if* the message has just been received (i.e. has the `ReceivedStamp` stamp).
+stamp *if* the message has just been received (i.e. has at least 1 `ReceivedStamp` stamp).
 You can create your own stamps by implementing :class:`Symfony\\Component\\Messenger\\Stamp\\StampInterface`.
 
+If you want to examine all stamps on an envelope, you can iterate through
```

Comment:
When not using a FQCN arguments, all stamps are returned grouped by their FQCN. 
I'm not sure it's worth mentioning it, but just "iterating" may be a bit misleading.

Reply:
Good point. I'll make some changes.